learning_resources_search/views.py

```diff
@@ -54,8 +54,12 @@ class LearningResourcesSearchView(ESView):
     def get(self, request):
         request_data = LearningResourcesSearchRequestSerializer(data=request.GET)
         if request_data.is_valid():
-            response = execute_learn_search(request_data.data)
-            return Response(SearchResponseSerializer(response).data)
+            extra_keys = set(request.GET.keys()) - set(request_data.fields.keys())
+            if extra_keys:
+                return Response(f"invalid parameters: {list(extra_keys)}", status=400)
+            else:
+                response = execute_learn_search(request_data.data)
+                return Response(SearchResponseSerializer(response).data)
```

Comment:
**Suggestion:** Since this is validation, should it go in the serializer? https://stackoverflow.com/a/36205600/2747370